components/elm/bld/namelist_files/namelist_definition.xml

```diff
@@ -1809,6 +1809,27 @@ Runtime flag to turn on/off PETSc based thermal model.
 Runtime flag to turn on/off variable soil thickness.
 </entry>
 
+<!-- ========================================================================================  -->
+<!-- Namelist options for downscaling from grid to topounit                                              -->
+<!-- ========================================================================================  -->
+
+<entry id="use_downscaling_to_tpu" 
+       type="logical" 
+       category="physics"
+       group="elm_inparm"
+       valid_values=".true.,.false." 
+       value=".false.">
+Runtime flag to turn on/off downscaling of atmosphric forcing from grid to topounit.
+</entry>
+<entry id="precip_downscaling" 
+       type="char*32" 
+       category="physics"
+       group="elm_inparm"       
+       valid_values="ERMM,FNM"
```

Comment:
It would be good to include a comment line here giving descriptions of what ERMM and FNM stand for

Reply:
@thorntonpe I have added comments in my latest commit as requested.